heed/src/env.rs

```diff
@@ -333,6 +335,49 @@ impl Drop for EnvInner {
     }
 }
 
+/// An helping function that transforms the LMDB types into Rust types (`MDB_val` into slices)
+/// and vice versa, the Rust types into C types (`Ordering` into an integer).
+extern "C" fn custom_key_cmp_wrapper<C: Comparator>(
+    a: *const ffi::MDB_val,
+    b: *const ffi::MDB_val,
+) -> i32 {
+    let a = unsafe { ffi::from_val(*a) };
+    let b = unsafe { ffi::from_val(*b) };
+    match C::compare(a, b) {
+        Ordering::Less => -1,
+        Ordering::Equal => 0,
+        Ordering::Greater => 1,
+    }
+}
+
+/// This is a special implementation of [`LexicographicComparator`] which means that no customized
+/// compare function is set to the underlying LMDB database instance. Its implementation of
+/// [`Comparator::compare`] will not be actually used.
+pub struct NoopComparator;
```

Comment:
I would like to change the name of this to something like `NoCustomComparator`, `IgnoreComparator`, `LexicograhicComparator` or `DefaultComparator` as it is in fact a type that is made to use the default LMDB lexicographic comparator one. Do you have better name ideas?

Reply:
I'll go with `DefaultComparator`; besides I add some more comprehensive comments to make sure most people won't get confused with it